
time="2025-08-04T15:50:18-03:00" level=warning msg="/home/rinha-user/rinha-de-backend-2025/participantes/Maxel-Uds-quarkus/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
 Network maxel-uds-quarkus_payment-router  Creating
 Network maxel-uds-quarkus_payment-router  Created
 Container api03  Creating
 Container api03  Created
 Container api01  Creating
 Container api02  Creating
 Container api01  Created
 Container api02  Created
 Container load_balancer  Creating
 Container load_balancer  Created
Attaching to api01, api02, api03, load_balancer
api03          | __  ____  __  _____   ___  __ ____  ______ 
api03          |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api03          |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api03          | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api03          | 2025-08-04 18:50:19,504 INFO  [io.quarkus] (main) payment-storage 1.0.0 native (powered by Quarkus 3.25.0) started in 0.026s. Listening on: http://0.0.0.0:8080
api03          | 2025-08-04 18:50:19,505 INFO  [io.quarkus] (main) Profile prod activated. 
api03          | 2025-08-04 18:50:19,505 INFO  [io.quarkus] (main) Installed features: [cdi, rest, rest-jackson, smallrye-context-propagation, vertx]
api01          | __  ____  __  _____   ___  __ ____  ______ 
api01          |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api01          |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api01          | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api01          | 2025-08-04 18:50:19,928 INFO  [io.quarkus] (main) payment-router 11.0.0 native (powered by Quarkus 3.11.1) started in 0.086s. Listening on: http://0.0.0.0:8080
api01          | 2025-08-04 18:50:19,928 INFO  [io.quarkus] (main) Profile prod activated. 
api01          | 2025-08-04 18:50:19,928 INFO  [io.quarkus] (main) Installed features: [cdi, rest, rest-client, rest-client-jackson, rest-jackson, smallrye-context-propagation, smallrye-fault-tolerance, vertx]
api02          | __  ____  __  _____   ___  __ ____  ______ 
api02          |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api02          |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api02          | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api02          | 2025-08-04 18:50:20,113 INFO  [io.quarkus] (main) payment-router 11.0.0 native (powered by Quarkus 3.11.1) started in 0.148s. Listening on: http://0.0.0.0:8080
api02          | 2025-08-04 18:50:20,114 INFO  [io.quarkus] (main) Profile prod activated. 
api02          | 2025-08-04 18:50:20,114 INFO  [io.quarkus] (main) Installed features: [cdi, rest, rest-client, rest-client-jackson, rest-jackson, smallrye-context-propagation, smallrye-fault-tolerance, vertx]
load_balancer  | 2025/08/04 18:50:28 [error] 7#7: *3 open() "/usr/local/openresty/nginx/html/purge-payments" failed (2: No such file or directory), client: 172.20.0.1, server: , request: "POST /purge-payments HTTP/1.1", host: "localhost:9999"
[Kload_balancer exited with code 0
api02          | 2025-08-04 18:51:35,704 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 20 thread(s) running with 0 task(s) waiting
api01          | 2025-08-04 18:51:35,711 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 20 thread(s) running with 0 task(s) waiting
api02          | 2025-08-04 18:51:40,704 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 20 thread(s) running with 0 task(s) waiting
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#50,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-17) Thread Thread[#57,executor-thread-17,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#49,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#42,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#52,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#48,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#55,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-16) Thread Thread[#56,executor-thread-16,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#54,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#45,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#46,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-18) Thread Thread[#58,executor-thread-18,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-20) Thread Thread[#60,executor-thread-20,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 2025-08-04 18:51:40,713 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#53,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,713 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#54,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,713 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#55,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-18) Thread Thread[#58,executor-thread-18,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#53,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,704 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#47,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 
api02          | 2025-08-04 18:51:40,704 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#44,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-19) Thread Thread[#59,executor-thread-19,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,705 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#51,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,704 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#43,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,707 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api02          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api02          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api02          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api02          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api02          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api02          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api02          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api02          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api02          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api02          | Caused by: java.lang.InterruptedException
api02          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api02          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api02          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api02          | 	... 13 more
api02          | 
api02          | 2025-08-04 18:51:40,707 INFO  [io.quarkus] (Shutdown thread) payment-router stopped in 10.009s
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#43,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#42,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 INFO  [io.qua.thread-pool] (Shutdown thread) Awaiting thread pool shutdown; 14 thread(s) running with 0 task(s) waiting
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#51,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#49,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#44,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-19) Thread Thread[#59,executor-thread-19,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-17) Thread Thread[#57,executor-thread-17,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#47,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-20) Thread Thread[#60,executor-thread-20,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#46,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,714 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#52,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,711 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,713 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#48,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,718 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#45,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,718 ERROR [org.jbo.thr.errors] (executor-thread-16) Thread Thread[#56,executor-thread-16,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,718 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#50,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:48)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$startProcessWorker$0(PaymentProcessorWorker.java:41)
api01          | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api01          | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api01          | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
api01          | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api01          | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api01          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api01          | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api01          | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api01          | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api01          | Caused by: java.lang.InterruptedException
api01          | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api01          | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api01          | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPaymentToProcess(PaymentProcessorWorker.java:46)
api01          | 	... 13 more
api01          | 
api01          | 2025-08-04 18:51:40,718 INFO  [io.quarkus] (Shutdown thread) payment-router stopped in 10.011s
[Kapi01 exited with code 143
api03          | 2025-08-04 18:51:41,801 INFO  [io.quarkus] (Shutdown thread) payment-storage stopped in 0.003s
[Kapi03 exited with code 143
log truncated at line 1000
